tests/runner/test_config.py

```diff
@@ -68,3 +71,29 @@ def test_dynamics_options():
         assert config_inp.integrator.lower().replace(
             "_", ""
         ) == d.integrator().__class__.__name__.lower().replace("integrator", "")
+
+
+# Skip on windows due to file permission issues
+# @pytest.mark.skipif(
+#    platform.system() == "Windows", reason="File permission issues on Windows"
+# )
+def test_logfile_creation():
+    # Test that the logfile is created by either the initialisation of the runner or of a config
+    with tempfile.TemporaryDirectory() as tmpdir:
+        # Load the demo stream file.
+        mols = sr.load(sr.expand(sr.tutorial_url, "merged_molecule.s3"))
+        from pathlib import Path
+
+        # Instantiate a runner using the default config.
+        # (All default options, other than platform="cpu".)
+        config = Config(output_directory=tmpdir, log_file="test.log")
+        assert config.log_file is not None
+        assert Path.exists(config.output_directory / config.log_file)
+
+        # Instantiate a runner using the default config.
+        # (All default options, other than platform="cpu".)
```

Comment:
This comment doesn't make sense here. (Must have been copied from another test.)